bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerEntry.java

```diff
@@ -53,22 +54,49 @@ public long getLength() {
         return length;
     }
 
+    /**
+     * Returns the content of the entry.
+     * This method can be called only once.
+     * While using v2 wire protocol this method will automatically release the internal ByteBuf
+     * @return
+     */
     public byte[] getEntry() {
+        if (data == null) {
```

Comment:
@eolivelli you don't have to revert it. it is good to make it clear. I just want to clarify to make things clear at this pull request. for assertion, I would prefer using guava Preconditions.checkState(...).